zephyr/cmake/mcuboot_zephyr_runner_mass_erase.cmake

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) 2022 Legrand North America, LLC.
```

Comment:
This isn't your copyright, this is a cut and paste of the original file, the copyright is `# Copyright (c) 2017 Open Source Foundries Limited`

Reply:
Thank you for pointing this out.  I missed the copyright in the file this was extracted from.  Now fixed.